packages/terra-modal/lib/Modal.scss

```diff
@@ -0,0 +1,31 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: #fff;
+  border-radius: 5px;
+  bottom: 10px;
+  display: block;
+  left: 10px;
+  max-height: 100%;
+  outline: none;
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
Let's add an `isScrollable` prop that adds the `overflow-y: auto` style when set to true. This allows us to handle both cases described above.